dev_guide/builds/build_strategies.adoc

```diff
@@ -525,3 +525,48 @@ spec:
     file to use, relative to the source `contextDir`.
     If `contextDir` is omitted, it defaults to the root of the repository.
     If `jenkinsfilePath` is omitted, it defaults to *_Jenkinsfile_*.
+
+[[jenkins-pipeline-strategy-environment]]
+=== Environment Variables
+
+To make environment variables available to the
+xref:../../architecture/core_concepts/builds_and_image_streams.adoc#pipeline-build[Pipeline build]
+process, you can add environment variables to the `jenkinsPipelineStrategy` definition
+of the `BuildConfig`.
+
+The environment variables defined there are set as parameters on the job in Jenkins associated with
+the `BuildConfig`. 
+
+For example:
+
+[source,yaml]
+----
+jenkinsPipelineStrategy:
+...
+  env:
+    - name: "FOO"
+      value: "BAR"
+
+----
+
+Specifics on the mapping between `BuildConfig` environment variables and Jenkins job parameters:
```

Comment:
I think it might work better if the above sentence was deleted, and the first two points were in paragraph form. They seem like new information about how the persistence of the en vars work. I think the last few points are more deserving of being in a list (see below point).